walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,14 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+    @Test
+    public void testGreetingTrimsWhitespace() {
```

Comment:
This function is supposed to trim whitespace, but your greet() function only removes the characters '@', '#', '$', '%'. Either fix the greet() function to trim whitespace or change the name/purpose of this function

P.S. I believe Java has a string function that automatically removes whitespace for you! Try that with the original test case

Reply:
Thank you zian for pointing this out! I didn't even notice the function name and the comment. The files are updated!